src/main/java/core/basesyntax/Colors.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+public enum Colors {

```

Comment:
Please, use singular form for ENUM names
```suggestion
public enum Color {
```